content/storage/bind-mounts.md

```diff
@@ -266,6 +266,38 @@ $ docker container stop devtest
 $ docker container rm devtest
 ```
 
+## Recursive mounts
+
+When you bind mount a path that itself contains mounts, those submounts are
+also included in the bind mount by default. This behavior is configurable,
+using the `bind-recursive` option for `--mount`.
+
+If the bind mount is read-only, the Docker Engine makes a best-effort attempt
+at making the submounts read-only as well. This is referred to as recursive
+read-only mounts. Recursive read-only mounts require Linux kernel version 5.12
+or later. If you're running an older kernel version, submounts are
+automatically mounted as read-write, even if you define the bind mount as
+`readonly`.
```

Comment:
ah, thanks!

Reply:
Hmm wait. I don't suppose `--mount type=bind,readonly` would cause an error. Are you referring to `--mount type=bind,readonly,bind-recursive=readable` or something?

Reply:
Yes, the two “readonly”s are confusing here 😅

Reply:
got it, thank you 👍🏻